assets/shaders/MenuShader.js

```diff
@@ -0,0 +1,98 @@
+Phaser.Filter.sunset = function (game) {
+
+    Phaser.Filter.call(this, game);
+
+    this.fragmentSrc = [
+        "precision highp float;",
+
+        "uniform float     time;",
+        "uniform vec2      resolution;",
+        "uniform vec2      mouse;",
+        
+    		"vec2 hash2(vec2 p ) {",
+    		   "return fract(sin(vec2(dot(p, vec2(123.4, 748.6)), dot(p, vec2(547.3, 659.3))))*5232.85324);",
+    		"}",
+    		"float hash(vec2 p) {",
+    		  "return fract(sin(dot(p*11.5, vec2(43.232, 75.876)))*45246.32957); ",  
+    		"}",
+
+    		"//Based off of iq's described here: http://www.iquilezles.org/www/articles/voronoilin",
+    		"float voronoi(vec2 p, float s) {",
+    		    "vec2 n = floor(p);",
+    		    "vec2 f = fract(p);",
+    		    "float md = 5.0;",
+    		    "vec2 m = vec2(0.0);",
+    		    "for (int i = -1;i<=1;i++) {",
+    		        "for (int j = -1;j<=1;j++) {",
+    		            "vec2 g = vec2(i, j);",
+    		            "vec2 o = hash2(n+g)*s;",
+    		            "o = 0.5+0.5*sin(0.64+5.038*o);",
+    		            "vec2 r = g + o - f;",
+    		            "float d = dot(r, r);",
+    		            "if (d<md) {",
+    		              "md = d;",
+    		              "m = n+g+o;",
+    		           " }",
+    		        "}",
+    		    "}",
+    		    "return md;",
+    		"}",
+
+    		"float ov(vec2 p) {",
+    		    "float v = 0.0;",
+    		    "float a = 0.4;",
+    		    "for (int i = 0;i<6;i++) {",
+    		        "v+= voronoi(p, 1.0)*a;",
+    		        "p*=2.0;",
+    		        "a*=0.5;",
+    		    "}",
+    		    "return v;",
+    		"}",
+
+    		"float TOD = 0.0;",
+    		"void main( void )",
+    		"{",
+    		    
+    		    "//adjust uv",
+    			"vec2 uv = gl_FragCoord.xy;",
+    		    "uv.x/=resolution.x;",
+    		    "uv.y/=resolution.y;",
+    		    "uv.x*=(resolution.x/resolution.y);",
+    		    
+    		    "//TOD calculation",
```

Comment:
I changed it anyways. Just so it doesn't clash when we are looking for our to do lists.